mrmustard/lab_dev/circuits.py

```diff
@@ -103,7 +103,11 @@ def draw(self, layout: str = "spring_layout", figsize: tuple[int, int] = (10, 6)
         Returns:
             A figure showing the tensor network.
         """
-        components = connect(self.components)
+        components = self.components
+        for component in components:
+            if component.wires.bra:
+                components = add_bra(components)
+        components = connect(components)
```

Comment:
```suggestion
                components = add_bra(components)
                break
        components = connect(components)
```

Reply:
Btw I'm leaving this as is for now, because I want to be able to test that we get the right result when we also have the bra part. But as soon as we add the ABC for the attenuator, I will change it so that if only kets are needed, the bra part is not added automatically

Reply:
ok, the break is just to save time